scripts/globals/conquest.lua

```diff
@@ -566,6 +566,61 @@ local function conquestRanking()
     return GetNationRank(xi.nation.SANDORIA) + 4 * GetNationRank(xi.nation.BASTOK) + 16 * GetNationRank(xi.nation.WINDURST)
 end
 
+
+xi.conquest.toggleRegionalNPCs = function(zone)
+    -- Show/Hide regional NPCs
+    -- If there is a draw or a 1st place Alliance, those NPCs won't be available.
+    local id = zone:getID()
+    if
+        id == xi.zone.PORT_BASTOK or
+        id == xi.zone.SOUTHERN_SAN_DORIA or
+        id == xi.zone.WINDURST_WOODS
+    then
+        -- When not a tie
+        if not IsConquestAlliance() then
+            local regionalNPCNames =
+            {
+                "Nokkhi_Jinjahl",
+                "Ominous_Cloud",
+                "Valeriano",
+                "Mokop-Sankop",
+                "Cheh_Raihah",
+                "Nalta",
+                "Dahjal"
+            }
+
+            -- TODO: Do we need to worry about beastmen's rank?
+            local rankings =
+            {
+                { GetNationRank(xi.nation.SANDORIA), xi.zone.SOUTHERN_SAN_DORIA },
+                { GetNationRank(xi.nation.BASTOK), xi.zone.PORT_BASTOK },
+                { GetNationRank(xi.nation.WINDURST), xi.zone.WINDURST_WOODS },
+            }
+
+            table.sort(rankings, function (a, b) return a[1] > b[1] end)
+
+            local firstPlaceZone = rankings[1][2]
+
+            if firstPlaceZone == zone:getID() then
+                print("Making regional conquest NPCs available in: " .. zone:getName())
+            end
+
+            for _, name in pairs(regionalNPCNames) do
+                local results = zone:queryEntitiesByName(name)
+                for _, entity in pairs(results) do
+                    -- Will be the real entity if it has X/Y/Z positions
+                    if math.abs(entity:getXPos()) > 0 then
+                        entity:setStatus(xi.status.DISAPPEAR)
```

Comment:
I don't think this condition will be hit to disable NPCs if there's a tie.  Might need to do that outside of the IsConquestAlliance() condition.